exec/java-exec/src/main/java/org/apache/drill/exec/work/filter/RuntimeFilterWritable.java

```diff
@@ -103,6 +103,27 @@ public RuntimeFilterWritable duplicate(BufferAllocator bufferAllocator) {
     return new RuntimeFilterWritable(runtimeFilterBDef, cloned);
   }
 
+  public void retainBuffers(final int increment) {
+    if (increment <= 0) {
+      return;
+    }
+    for (final DrillBuf buf : data) {
+      buf.retain(increment);
+    }
+  }
+
+  public RuntimeFilterWritable newRuntimeFilterWritable(BufferAllocator bufferAllocator) {
```

Comment:
Please add a TODO comment saying: Not used currently because of <DRILL-XXXX> reference the JIRA number which you will create with explanation.

Reply:
Ok